### DISCUSSION TITLE
Internal Server Error after Installing Demo Templates

### DISCUSSION THREAD

**Discussion by prager** [solspace/craft-freeform#416] [Questions & Answers] [answered]
This is a fatal error for Craft Pro 4.2.0.2 with Freeform 4.0.0-beta.18(trial)

After I installed demo templates I can no longer edit a new form via Freeform dashboard. The form creates OK, but when I click on it and attempt to edit the form I get Internal Server Error. The same error is thrown when clicking on Settings->Success Templates. It may be because I created and set a success template prior to demo templates installation.

Settings->Diagnostics indicate that "Success Templates Directory Path: Not set correctly". It shows the path to previously set success template that was created in custom directory as "templates/forms/success.twig". 

Is there out there any other remedy than reinstall the plugin? Thank you in advance for any advice.

![err-craft_internal](https://user-images.githubusercontent.com/18473082/182666125-63f01d8a-3581-4084-9da1-5f5527e7d5eb.png)

**Comment by kjmartens** [comment, marked as answer]
I have responded to the duplicate issue reported  in #417
